toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -212,34 +282,63 @@ func (im *IsoMaker) setUpIsoGrub2Bootloader() {
 		efiBootImgTempMountDir, // Path to mount the image to.
 	}
 	logger.Log.Debugf("Mounting '%s' to '%s' to copy EFI modules required to boot grub2.", im.efiBootImgPath, efiBootImgTempMountDir)
-	shell.MustExecuteLive("mount", mountArgs...)
+	err = shell.ExecuteLive(false /*squashErrors*/, "mount", mountArgs...)
+	if err != nil {
+		return err
+	}
 
 	defer func() {
 		logger.Log.Debugf("Unmounting '%s'.", efiBootImgTempMountDir)
-		logger.PanicOnError(syscall.Unmount(efiBootImgTempMountDir, 0), "Failed to unmount '%s'.", efiBootImgTempMountDir)
+		tmpErr := syscall.Unmount(efiBootImgTempMountDir, 0)
+		if tmpErr != nil {
+			if err != nil {
+				err = fmt.Errorf("Failed to unmount '%s'.\nError:%v\nClean-up Error:%v", efiBootImgTempMountDir, err, tmpErr)
```

Comment:
nit: `"failed to unmount '%s':\n%w:\nclean-up error: %w"`

Reply:
fixed.